git-work.sh

```diff
@@ -31,47 +31,46 @@ usage() {
   echo "Available subcommands are:"
   echo "   issue <id>              Start a new branch from issue <id>"
   echo "   commit <\"msg\">          Commit current work using <message>"
-  echo "   done <id>               Finish current work and merge"
-  echo "   push <master>           Push to the server"
+  echo "   done                    Finish current work and merge"
+  echo "   push                    Push to the server"
   echo "   release <tag> [<\"msg\">] Release a new tag called <tag> with <message>"
   echo "   pr <id>                 Create a branch from a pull-request"
 }
 
-if [ -z $1 ] || [ -z $2 ]; then
-  usage
-  exit 1
-fi
-
 case "$1" in
   # start a new branch
   issue)
+    [ -z $2 ] && ( usage && exit 1 )
     git checkout -b "$2"
     ;;
   # commit current work
   commit)
+    [ -z $2 ] && ( usage && exit 1 )
```

Comment:
I've got this error: `/git-work.sh: line 48: [: first: binary operator expected`